.taskcluster.yml

```diff
@@ -17,8 +17,7 @@ tasks:
       deadline: {$fromNow: '2 hours'}
       provisionerId: aws-provisioner-v1
       workerType: github-worker
-      routes:
```

Comment:
Do you want notification on pull requests too, @rpappalax ?

Reply:
@JohanLorenzo  for now, we just want additional notifications whenever the UI tests complete.  I added that in to the python script here: https://github.com/mozilla-mobile/focus-android/blob/master/tools/taskcluster/schedule-master-build.py#L101  hopefully the scopes allow notify from python client.

Reply:
Okay, since this type of job doesn't use `schedule-master-build.py`, we don't need to expose this scope on PRs.